dblp-aliases.csv

```diff
@@ -1722,6 +1722,7 @@ Marco G. Valtorta,Marco Valtorta
 Marco A. Wiering,Marco Wiering
 Marco Antonio Zúñiga Zamalloa,Marco Zuniga
 Marco Zúñiga,Marco Zuniga
+Mao Rui 0001,Mao Rui

```

Comment:
Please delete this alias (the DBLP entry does not have any aliases for `Rui Mao 0001`).